src/API/Logs/Severity.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+declare(strict_types=1);
+
+namespace OpenTelemetry\API\Logs;
+
+class Severity
```

Comment:
Now we are in 8.1, do you have any thoughts on making something like this an enum? :thinking:

Reply:
great suggestion, now it's an enum. Currently, `LogRecord` accepts int|enum, but I'm certainly open to accepting just the enum.

Reply:
I can't see a concrete restriction on the  SeverityNumber (see: [data model](https://github.com/open-telemetry/opentelemetry-specification/blob/v1.32.0/specification/logs/data-model.md#field-severitynumber)), but we could potentially restrict to known values only `Severity::tryFrom($value)` etc.

In that case, maybe internally we could work with an enum throughout for this new functionality? :thought_balloon:

Reply:
Let's try it out. I've updated to only use an enum.